### DISCUSSION TITLE
Add option to simultaneously calibrate offset for all difficulties

### DISCUSSION THREAD

**Discussion by JamieWright99** [ppy/osu#24194] [Ideas] [answered]
For a beatmap with a large amount of difficulties it would be nice to be able to change all of the offsets with a single play. Currently you can only change the individual difficulties offset, but since most maps use the same timing for all difficulties it makes sense to apply the offset to all diffs. Here's a rough idea of how it could be implemented, but another idea is that it could also be a toggle option in the settings.
![image](https://github.com/ppy/osu/assets/102170503/e56cce57-419a-4734-9ed1-a629e65b746b)
Apologies if this has already been discussed! I tried searching and didn't see anything.

**Comment by peppy** [comment]
I think it always makes sense to calibrate all difficulties as 99.9999% of the time they should share all timing. So let's just change it to that.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/24290
